onadata/libs/serializers/floip_serializer.py

```diff
@@ -78,14 +79,38 @@ def parse_responses(responses, session_id_index=SESSION_ID_INDEX,
     yield submission
 
 
+class UUIDField(serializers.CharField):
+    """
+    Custom UUIDField that stores the values of a UUID as hex but
+    represents the as a complete UUID4 string
+    """
+    def to_representation(self, obj):
+        return str(uuid.UUID(obj))
+
+    def to_internal_value(self, obj):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/manual/Ona/onadata/pullRequest?prid=4906603)